configs/animal_2d_keypoint/topdown_heatmap/README.md

```diff
@@ -1,7 +1,53 @@
 # Top-down heatmap-based pose estimation
 
-Top-down methods divide the task into two stages: object detection and pose estimation.
+Top-down methods divide the task into two stages: object detection, followed by single-object pose estimation given object bounding boxes Instead of estimating keypoint coordinates directly, the pose estimator will produce heatmaps which represent the
+likelihood of being a keypoint, following the paradigm introduced in [Simple Baselines for Human Pose Estimation and Tracking](http://openaccess.thecvf.com/content_ECCV_2018/html/Bin_Xiao_Simple_Baselines_for_ECCV_2018_paper.html).
 
-They perform object detection first, followed by single-object pose estimation given object bounding boxes.
-Instead of estimating keypoint coordinates directly, the pose estimator will produce heatmaps which represent the
-likelihood of being a keypoint.
+<div align=center>
+<img src="https://user-images.githubusercontent.com/15977946/146522977-5f355832-e9c1-442f-a34f-9d24fb0aefa8.png" height=400>
+</div>
+
+## Results and Models
+
+### Animal-Pose Dataset
+
+Results on AnimalPose validation set (1117 instances)
+
+|   Model    | Input Size |  AP   |  AR   |                   Details and Download                    |
+| :--------: | :--------: | :---: | :---: | :-------------------------------------------------------: |
+| HRNet-w32  |  256x256   | 0.740 | 0.780 |  [hrnet_animalpose.md](./animalpose/hrnet_animalpose.md)  |
```

Comment:
HRNet32 performance is abnormally high and unaligned with [master](https://github.com/open-mmlab/mmpose/blob/master/configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.md). Please double check it with @Ben-Louis

Reply:
I just run the test for this setting with master and dev-1.x, respectively. The results match this table but are different from [the record](https://github.com/open-mmlab/mmpose/blob/master/configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.md).

Reply:
Ok, then let's keep it this way in this PR. Meanwhile, @liqikai9 please retain the hrnet32 and hrnet48 models to further verify the performance.